File: MUJAHID/c++/Overriding.cpp
#include<iostream>
using namespace std;
class A
{
    public:
    void show(){
    cout<<"Class A"<<endl;
}
};
class B:public A
{
    public:
    void show(){
    cout<<"Class B"<<endl;
}
};
int main()
{
    B b;
    b.show();
    return 0;
}

File: java/Overriding.java
class Demo{
    public void show(){
        System.out.println("Demo");
    }
}

public class Overriding  extends Demo{
    public void show1() {
        System.out.println("hello");
    }

    public static void main(String[] args) {
        Overriding overriding = new Overriding();
        overriding.show();
    }

}
